admin/users/serializers.py

```diff
@@ -0,0 +1,22 @@
+"""
+Serialize user
+"""
+# from admin.nodes.serializers import serialize_simple_user
+
+
+def serialize_user(user):
+    return {
+        'name': user.fullname,
+        'nodes': map(serialize_simple_node, user.contributor_to),
```

Comment:
Just FYI, with the backref removal user.contributor_to calls User.contributed, which does a query of all Nodes with the user id in their contributors ForeignField(list=True), then it runs through that entire list checking is_deleted and is_dashboard, then you're looping through it again with map. Not saying you shouldn't, just saying this could be a possible bottleneck/performance optimization opportunity.